platforms/hermes-kotlin/src/main/kotlin/ai/snips/hermes/Ontology.kt

```diff
@@ -137,7 +137,7 @@ sealed class SessionTermination(val type: SessionTermination.Type) {
     object Nominal : SessionTermination(SessionTermination.Type.NOMINAL)
     object SiteUnAvailable : SessionTermination(SessionTermination.Type.SITE_UNAVAILABLE)
     object AbortedByUser : SessionTermination(SessionTermination.Type.ABORTED_BY_USER)
-    object IntenNotRecognized : SessionTermination(SessionTermination.Type.INTENT_NOT_RECOGNIZED)
+    object IntentNotRecognized : SessionTermination(SessionTermination.Type.INTENT_NOT_RECOGNIZED)
```

Comment:
shoudl let the typoed one with a depecation warning on it. this way this is not a breaking change

Reply:
Well. this is kinda problematic when converting from/to C. 
Since both `Inten`**t**`NotRecognized` and `IntenNotRecognized` will map to `CSessionTermination.INTENT_NOT_RECOGNIZED` ...

What happens during a roundtrip is that : 
IntenNotRecognized -> INTENT_NOT_RECOGNIZED -> Inten**t**NotRecognized. 

a bit trickier than expected.